addons/product/models/product_attribute.py

```diff
@@ -227,7 +227,6 @@ class ProductTemplateAttributeValue(models.Model):
     attribute_id = fields.Many2one(
         'product.attribute', string='Attribute',
         related="product_attribute_value_id.attribute_id")
-    sequence = fields.Integer('Sequence', related="product_attribute_value_id.sequence")
```

Comment:
I'm doing the same change in my PR but then order has to be updated to `_order = 'attribute_line_id, product_attribute_value_id, id'` especially since you add a sequence on the line.